test/Examples/main.c

```diff
@@ -0,0 +1,14 @@
+#include "runtime/main.h"
+#include "runtime/stop.h"
+
+/* main entry into PROGRAM */
+void _QQmain();
+
+/* C main stub */
+int main(int argc, char **argv, char **envp)
+{
+  RTNAME(ProgramStart)(argc, argv, envp);
+  _QQmain();
+  RTNAME(ProgramEndStatement)();
```

Comment:
I think Peter's design is that the fortran program be called `main` and that the compiler generates the `ProgramStart`/`ProgramEndStatment`. But that also seems fine to start like this for tests.

Reply:
There are trade-offs to consider, but the stub is just to get something limping along for the moment.